components/Homepage/RoundActionCard/RoundActionCard.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  Center,
+  HStack,
+  Heading,
+  Spinner,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { Phase } from "services/PhaseMgmtService";
+
+interface Props {
+  phase: Phase;
+  roundId: number;
+  isAuthed?: boolean;
+  hasSignedUp?: boolean;
+  hasVoted?: boolean;
+  hasSubmitted?: boolean;
+  onProfile: () => void;
```

Comment:
nit - can we pass this as 'roundActionFunctions' so the top-layer signature is less verbose?